dmriprep/utils/vectors.py

```diff
@@ -375,3 +375,50 @@ def bvecs2ras(affine, bvecs, norm=True, bvec_norm_epsilon=0.2):
         rotated_bvecs[~b0s] /= norms_bvecs[~b0s, np.newaxis]
         rotated_bvecs[b0s] = np.zeros(3)
     return rotated_bvecs
+
+
+def reorient_vecs_from_ras_b(rasb_file, affines, b0_threshold=B0_THRESHOLD):
```

Comment:
Can we make it a method of the vectors object (and therefore, it will only take the ``affines`` parameter)? Also drop the ``b0_threshold`` argument, since that should set only once in the vectors class.

Reply:
yes we can